.readthedocs.yaml

```
@@ -11,11 +11,13 @@ build:
   os: ubuntu-20.04
   tools:
     python: "3.11"
+  apt_packages:
+    - graphviz
 
 python:
   install:
     - requirements: requirements.txt
 
 formats:
     - pdf
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)